src/styles/blocks/button.scss

```
@@ -0,0 +1,27 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
+  background-color: $blue_accent;
+  border-radius: 5px;
+  color: $white;
+  transition: background 0.3s, color 0.3s;
+
+  &--txt {
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    border: solid $blue_accent 1px;
+    transform: $transform;
+  }
+
+  &:hover {
+    color: $blue_accent;
+    background-color: $white;
+    border: solid $blue_accent 1px;
+    box-sizing: border-box;
+  }
```

```suggestion
  &:hover {
    color: $blue_accent;
    background-color: $white;
    border-color: $blue_accent;
  }
```
- you should change only border-color on hover instead of the border and in this case, you can apply the transition to it too
- box-sizing: border-box; you could apply to all elements with `*` selector, not only for `button:hover`

---

Not fixed